WooCommerce/WooCommerceTests/Tools/WeightFormatterTests.swift

```diff
@@ -5,20 +5,34 @@ final class WeightFormatterTests: XCTestCase {
 
     func test_weightFormat_returns_expected_values() {
```

Comment:
Just a note that these tests fail when using an Arabic simulator. Not sure if we want to fix this right now or as a debt later. 

![image](https://user-images.githubusercontent.com/198826/116456732-cf483300-a81f-11eb-995e-94a07ea369ef.png)

Reply:
I accidentally left my simulator in Spanish yesterday on a different PR and a dozen tests were failing because of that. I think we should address the issue separately. I see at least #2382 and #2132 related to this. The easiest solution is probably to force English/US as the System Language/Region in the test plan configuration.